app/views/posts/index.html.erb

```diff
@@ -1 +1,2 @@
-<h1>You are in index page of posts_index</h1>
+<%= render @user%>
```

Comment:
As shown in the wireframe provided you are to use `post.recent_comments` to render 5 most recent comments you can write something like this 
````
 <% post.recent_comments.each do |comment| %>
  <p><%= comment.author.name %>: <%= comment.text %></p> 
 <% end %>
````
![image](https://github.com/bilalrajput09/Blog_app_rails/assets/52497246/9f3ff997-ca63-44e8-a14d-2f80040b2ef9)

Reply:
Fixed